components/ai_chat/resources/page/components/page_context_toggle/index.tsx

```diff
@@ -54,16 +47,14 @@ function PageContextToggle() {
                 <SiteTitle size='small' />
               </div>
             </div>
-            <div
-              onMouseOver={() => setShowTooltip(true)}
-              onMouseOut={() => setShowTooltip(false)}
-              onTouchStart={handleInfoIconTouchStart}
-              onTouchEnd={handleInfoIconTouchEnd}
-              onFocus={() => setShowTooltip(true)}
-              onBlur={() => setShowTooltip(false)}
+            <Button
```

Comment:
Sorry to nitpick but I'm surprised the `preventDefault()` works on the Button - it shouldn't be an HTML event, but instead the wrapped Svelte event

Reply:
ok, so i have made the tooltip controlled which eliminates the need for preventDefault() and has a better UX as tapping the info icon should toggle visibility on mobile